static/app/views/replays/deadRageClick/deadClickList.tsx

```diff
@@ -0,0 +1,119 @@
+import {browserHistory, RouteComponentProps} from 'react-router';
+import styled from '@emotion/styled';
+
+import Alert from 'sentry/components/alert';
+import DatePageFilter from 'sentry/components/datePageFilter';
+import EnvironmentPageFilter from 'sentry/components/environmentPageFilter';
+import * as Layout from 'sentry/components/layouts/thirds';
+import PageFilterBar from 'sentry/components/organizations/pageFilterBar';
+import PageFiltersContainer from 'sentry/components/organizations/pageFilters/container';
+import {PageHeadingQuestionTooltip} from 'sentry/components/pageHeadingQuestionTooltip';
+import Pagination from 'sentry/components/pagination';
+import ProjectPageFilter from 'sentry/components/projectPageFilter';
+import SentryDocumentTitle from 'sentry/components/sentryDocumentTitle';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import useDeadRageSelectors from 'sentry/utils/replays/hooks/useDeadRageSelectors';
+import useOrganization from 'sentry/utils/useOrganization';
+import SelectorTable from 'sentry/views/replays/deadRageClick/selectorTable';
+import {
+  DeadRageSelectorItem,
+  DeadRageSelectorQueryParams,
+} from 'sentry/views/replays/types';
+
+function getAriaLabel(str: string) {
```

Comment:
What input are we expecting for `str` when it's an aria label?

Reply:
i'm parsing the dom element to get the aria label!